source/_redirects

```diff
@@ -1893,7 +1892,6 @@
 /components/tplink /integrations/tplink
 /components/tplink_lte /integrations/tplink_lte
 /components/traccar /integrations/traccar
-/components/trackr /integrations/trackr
```

Comment:
Please add a redirect under removed integrations.

https://github.com/home-assistant/home-assistant.io/blob/8b846e5bb1489ad576d050aca788397a02255c0c/source/_redirects#L2210

Reply:
Ahh missed that.  Done.